### ISSUE TITLE
2 Feature requests for Sync intervals 

### ISSUE THREAD

**Issue by saitoh183** [trakt/Plex-Trakt-Scrobbler#216]
When setting the sync Interval in multiuser configuration page or in the trakt channel setting page, It should show the time till next sync. 

Ability to set a specific time

**Comment by fuzeman** [comment]
> When setting the sync Interval in multiuser configuration page or in the trakt channel setting page, It should show the time till next sync. 

Good idea, this shouldn't be too hard to implement in the new configuration site.

Can't really add this to the trakt channel settings *(plex framework limitation)*, might be able to display this in the "Sync" menu though.

> Ability to set a specific time

Supporting custom [CRON expressions](https://en.wikipedia.org/wiki/Cron#CRON_expression) might be the easiest way to support this, will try look into this next week.
